lua/nvim-tree/actions/node/open-file.lua

```diff
@@ -211,11 +211,16 @@ local function open_in_new_window(filename, mode, win_ids)
     return
   end
 
-  local create_new_window = #vim.api.nvim_list_wins() == 1
+  local win_list = vim.api.nvim_list_wins()
+  local create_new_window = #win_list == 1 -- This implies that the nvim-tree window is the only one
   local new_window_side = (view.View.side == "right") and "aboveleft" or "belowright"
 
-  -- Target is invalid or window does not exist in current tabpage: create new window
-  if not vim.tbl_contains(win_ids, target_winid) then
+  -- Target is invalid: create new window
+  if not vim.tbl_contains(win_list, target_winid) then
+    vim.notify(
+      "That window doesn't exist, either this is a bug in the picker or it was just deleted",
```

Comment:
I don't think we need this; the fallback is sufficient.

Reply:
The silent fallback was actually quite annoying when I was trying to figure this out, and as I said, I don't see how we reach this case without a bug in the picker or some very exceptional circumstance. I was even wondering if we should call the picker again (for the case where the window somehow gets deleted between picking and using)

Reply:
I understand, however users are very sensitive about notifications. There might be a use case that we are not aware of.

> I was even wondering if we should call the picker again (for the case where the window somehow gets deleted between picking and using)

Let's keep it simple. If it becomes a problem we can revisit.